run

```diff
@@ -1,5 +1,50 @@
 #!/bin/bash
 
+# DKIM config
+dkimConfig()
+{
+  if [ ! -z "$OPENDKIM_DOMAINS" ] ; then
+    postconf -e milter_protocol=2
+    postconf -e milter_default_action=accept
+    postconf -e smtpd_milters=inet:localhost:12301
+
+    rm -f /etc/opendkim/KeyTable
+    rm -f /etc/opendkim/SigningTable
+
+    set $OPENDKIM_SELECTORS
+    for d in $OPENDKIM_DOMAINS ; do
+      if [ -z "$1" ] ; then
+        selector="mail"
+      else
+        selector="$1"
+      fi
+      DIR="/etc/opendkim/keys/$d"
```

Comment:
I don't think it is an issue.
With lastest version I just commited, creating multiple selectors per domain works, but usage is tied to the SigningTable file. So support for this would be tricky I think.

Reply:
Sorry it was a while since i messed around with DKIM. What would be the problem with a single SigningTable and multiple selectors for the same domain? each line seems to have both domain and selector hmm